tools/theme/components/button.ts

```diff
@@ -35,6 +35,7 @@ const Button: ComponentSingleStyleConfig = {
       color: colorScheme === 'primary' ? 'primary' : `texticon.${colorScheme}`,
       _hover: {
         bg: `background.${colorScheme === 'primary' ? 'interactive' : colorScheme}`,
+        textDecoration: 'underline',
```

Comment:
I think the ticket might be inaccurate - could you please remove this change? We'll address it in the future.